src/main/java/com/cinema/controllers/CustomGlobalExceptionHandler.java

```diff
@@ -0,0 +1,35 @@
+package com.cinema.controllers;
+
+import java.util.Date;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import org.springframework.context.support.DefaultMessageSourceResolvable;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+@ControllerAdvice
+public class CustomGlobalExceptionHandler extends ResponseEntityExceptionHandler {
+
+    protected ResponseEntity<Object> handleArgumentNotValid(
+            MethodArgumentNotValidException exception,
+            HttpHeaders headers, HttpStatus status,
+            WebRequest request) {
+        Map<String, Object> body = new LinkedHashMap<>();
+        body.put("timestamp", new Date());
+        body.put("status", status.value());
+        List<String> errors = exception.getBindingResult()
+                .getFieldErrors()
```

Comment:
does it work with password matching validator?
send a screenshot of response(from postman) for this case

Reply:
![Screenshot_1](https://user-images.githubusercontent.com/64158673/97693898-80a52080-1aaa-11eb-9862-77f646f3d340.png)

Reply:
fixed)